src/Header/Header.js

```diff
@@ -16,18 +17,35 @@ const getDayTextStyles = (numberOfDays) => {
   };
 };
 
+const DayNumber = ({ day, textStyle, dayColor }) => (
+  <Text style={[textStyle, styles.dayNumber, { color: dayColor }]}>
+    {getFormattedDate(day, 'D')}
+  </Text>
+);
+
 const Column = ({ column, numberOfDays, format, style, textStyle }) => {
+  const { dayBG, dayColor } =
+    moment().startOf('day').diff(moment(column), 'days') === 0 &&
+    format === 'ddd+'
```

Comment:
Sorry sorry tiny nitpick: could you move the && to the front of this line instead of the back of the previous line? I think that makes the ternary clearer bc rn it looks like "format === ddd+" is the only thing in the expression.

Feel free to disagree though, there could definitely be an argument in the other way that I haven't considered.

Reply:
The prettier was enforcing that through eslint, so I just disabled it.